check_line_length.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+import glob
+import os
+import re
+import sys
+
+LIMIT = 75
+
+
+def check_url(line):
+    """
+    Check if there is a url present in the given line
+    """
+
+    url = \
+        re.findall('http[s]?://(?:[a-zA-Z]|[0-9]|[$-_@.&+]|[!*\(\), ]|(?:%[0-9a-fA-F][0-9a-fA-F]))+'
+                   , line)
+    if url:
+        return True
+    else:
+        return False
```

Comment:
simplify this if/else block to just:

```python
return bool(url)
```

Which will return either `True` or `False` depending if the result is not empty or empty respectively.